reporter/src/main/kotlin/reporters/WebAppReporter.kt

```diff
@@ -45,10 +50,13 @@ class WebAppReporter : Reporter {
 
         val outputFile = outputDir.resolve(reportFilename)
 
-        outputFile.bufferedWriter().use {
-            it.write(prefix)
-            evaluatedModel.toJson(it, prettyPrint = false)
-            it.write(suffix)
+        FileOutputStream(outputFile, true).use {
```

Comment:
I don't think that `append = true` is correct here anymore.

Reply:
Here's what I'd propose:

    outputFile.outputStream().use { stream ->
        val base64Stream = Base64.getEncoder().wrap(stream)
        val gzipWriter = GzipCompressorOutputStream(base64Stream).bufferedWriter()

        stream.writer().use { writer ->
            writer.write(prefix)
            evaluatedModel.toJson(gzipWriter, prettyPrint = false)
            writer.write(suffix)
        }
    }

Reply:
I have reverted that change again, because it only seemed to work because of the wrong `append = true`.